at_secondary/at_secondary_server/config/config.yaml

```diff
@@ -123,4 +123,9 @@ sync:
   # Represents the size of buffer for each batch.
   bufferSize: 5242880
   # Represents the numbers of keys to send in each batch.
-  pageLimit: 25
\ No newline at end of file
+  pageLimit: 25
+
+#set to true in testing mode, false by default
+#IMPORTANT NOTE : please set testingMode to true only if you know what you're doing. Set to false when not testing
+testing:
+  testingMode: true
```

Comment:
It should also be false here - this is the file we push to all secondary servers.